README.md

```diff
@@ -251,13 +251,13 @@ from
         select
             x.wal,
             x.is_rls_enabled,
-            x.subscription_ids,
+            x.subcription_ids,
             x.errors
         from
             realtime.apply_rls(data::jsonb) x(wal, is_rls_enabled, subcription_ids, errors)
```

Comment:
```sql
realtime.apply_rls(data::jsonb) x(wal, is_rls_enabled, subcription_ids, errors)
```
how about changing the alias here from `subcription_ids` to `subscription_ids` instead